fw.lua

```diff
@@ -233,14 +233,20 @@ local function _parse_collection(self, collection, opts)
 				_collection[n] = value
 			end
 			return _collection
+		end,
+		multi = function(self, collection, value)
+			_log(self, "_parse_collection is getting multi values: " .. table.concat(value, ','))
+			local n = 0
+			local _collection = {}
+			for _, val in ipairs(value) do
+				n = n + 1
+				_collection[n] = collection[val]
+			end
+			return _collection
 		end
 	}
 
-	if (type(collection) ~= "table") then
-		return collection
-	end
-
-	if (opts == nil) then
+	if (type(collection) ~= "table" or opts == nil) then
```

Comment:
Again, please keep commits atomic. This change is unrelated to the context of the pull request.